### PR TITLE
samples: add simple

### PR DESCRIPTION
**Pull request by wilu-nordic**

### CODE REVIEW

File: samples/simple/README.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/simple`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+|test_sample|
```
Comment (annakielar) [inline samples/simple/README.rst:85]:
```suggestion
=======

|test_sample|
```

File: samples/simple/README.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+#. |connect_kit|
+#. |connect_terminal|
```
Comment (annakielar) [inline samples/simple/README.rst:88]: Just to confirm - is that the last step for testing? Any sample-specific steps?

File: samples/simple/src/main.c [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+	npmx_instance_t *npmx_instance = npmx_driver_instance_get(pmic_dev);
+
+	/* Get pointer to charger instance. */
+	npmx_charger_t *charger_instance = npmx_charger_get(npmx_instance, 0);
+
+	/* Register callback for VBUSIN thermal event used for detecting CC lines status. */
+	npmx_core_register_cb(npmx_instance, vbusin_thermal_callback,
+			      NPMX_EVENT_GROUP_VBUSIN_THERMAL);
+
+	/* Enable detecting CC status changed. */
```
Comment (annakielar) [inline samples/simple/src/main.c:74]:
```suggestion
	/* Enable detecting if CC status changed. */
```

**Review by annakielar — CHANGES_REQUESTED** [review]

File: samples/simple/README.rst (hunk 2 above)
Comment (wilu-nordic) [reply samples/simple/README.rst:88]: The last step is to look at the terminal output. Should it be described here?

File: samples/simple/README.rst [hunk 4]
```diff
@@ -0,0 +1,94 @@
+.. _simple_sample:
+
+Simple
+######
+
+.. contents::
+   :local:
+   :depth: 2
+
+The simple sample demonstrates how to setup CHARGER, LEDs and USB CC events with npmx drivers.
```
Comment (annakielar) [inline samples/simple/README.rst:10]:
```suggestion
The Simple sample demonstrates how to perform basic activities on CHARGER, LEDs and USB CC events with npmx drivers.
```

File: samples/simple/README.rst [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also requires an nPM1300 EK.
+
+Overview
+********
+
+This sample uses drivers to do the following:
```
Comment (annakielar) [inline samples/simple/README.rst:24]:
```suggestion
This sample uses npmx drivers to do the following:
```

File: samples/simple/README.rst (hunk 4 above)
Comment (annakielar) [reply samples/simple/README.rst:10]: Additionally, in this sentence we mentioned 3 of 4 activities from Overview section - should the fourth one be mentioned somehow?

File: samples/simple/README.rst (hunk 2 above)
Comment (annakielar) [reply samples/simple/README.rst:89]: Yes, please mention that activity and what result you expect, e.g. #. Check if there are any errors in the terminal.

**Review by annakielar — CHANGES_REQUESTED** [review]

File: samples/simple/README.rst (hunk 4 above)
Comment (wilu-nordic) [reply samples/simple/README.rst]: Added information about ADC.

**Review by annakielar — CHANGES_REQUESTED** [review]

**Review by annakielar — DISMISSED** [review]

**Review by annakielar — APPROVED** [review]

**Review by masz-nordic — APPROVED** [review]

**Pull request merged**